common/utils.go

```diff
@@ -115,10 +117,17 @@ func ForwardHeadersByPrefix(p string, from http.Header, to http.Header) {
 
 // ErrorLogEncoder decorates the errorEncoder in such a way that
 // errors are logged with their corresponding unique request identifier
-func ErrorLogEncoder(logger kitlog.Logger, ee kithttp.ErrorEncoder) kithttp.ErrorEncoder {
-	var errorLogger = logging.Error(logger)
+/*pass get logger func to pull logger out and use in the func returned; basically like the other ones */
```

Comment:
This will be part of the go documentation.  Is the plan to remove this comment?

Reply:
Yes! Sorry, I'll get in the habit of better note tracking.